tasks/configuration.yml

```diff
@@ -49,4 +49,4 @@
   with_dict: "{{ nginx_stream_configs }}"
   notify:
    - reload nginx
-  when: nginx_official_repo_mainline
+  when: nginx_stream_params is defined and nginx_stream_configs is defined
```

Comment:
`nginx_stream_params` and `nginx_stream_configs` are always defined (they are defined in `defaults/main.yml`). 
We have to check if there are not an empty list for `nginx_stream_params `or not an empty map for `nginx_stream_configs`